### ISSUE TITLE
How can i get pretrained model?

### ISSUE THREAD

**Issue by hoya-cho** [SwinTransformer/Swin-Transformer-Object-Detection#190]
How can i get pretrained model Cascade R cnn + swin transformer?

Mostly, there is only the cascade mask rcnn pre-training model, and the cascade rcnn + swin transformer pre-training model cannot be found. Where can I get it?

So, instead of training the cascade mask rcnn pre-learning model with cascade rcnn+ swin transformer, it seems that additional training is not possible because the weight parameters are different, right?
